index.js

```diff
@@ -295,7 +299,12 @@ const meow = (helpText, options = {}) => {
 		({description} = package_);
 	}
 
-	help = (description ? `\n  ${description}\n` : '') + (help ? `\n${help}\n` : '\n');
+	// Change to `&&=` when targeting Node 15+
+	if (description) {
+		description = help ? `\n  ${description}\n` : `\n${description}`;
+	}
```

Comment:
Node 15+ supports [logical AND assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND_assignment#browser_compatibility), which makes this simpler:

```js
description &&= help ? `\n  ${description}\n` : `\n${description}`;
```